CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ add_subdirectory(3rd-party)
 find_package(Qt5Core REQUIRED)
 find_package(Qt5Network REQUIRED)
 
-execute_process(COMMAND git describe
+execute_process(COMMAND git describe --dirty
```

Comment:
Won't this be wrong when the version is not dirty (i.e. an exact tag)?

Reply:
No, see [`man git-describe`](https://git-scm.com/docs/git-describe#git-describe---dirtyltmarkgt)

Reply:
Hmm, my earlier comment seems to have been lost. Anyway, I just said you were right, that I had seen the man but had overlooked that bit.